Project.toml

```diff
@@ -8,9 +8,12 @@ BlockDiagonals = "0a1fb500-61f7-11e9-3c65-f5ef3456f9f0"
 ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
 FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
 FiniteDifferences = "26cc04aa-876d-5657-8c51-4c34ba976000"
+JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
```

Comment:
We should add all test dependencies in the `[extras]` section of the Project.toml and in the [targets]
You can see the example of `ChainRulesTestUtils`

Reply:
Will do.